lib/logflare/source/recent_logs_server.ex

```diff
@@ -237,7 +240,7 @@ defmodule Logflare.Source.RecentLogsServer do
 
   def terminate(reason, state) do
     # Do Shutdown Stuff
-    Logger.info("Going Down - #{inspect(reason)} - #{state.source_id}", %{
+    Logger.error("Going Down - #{inspect(reason)} - #{state.source_id}", %{
```

Comment:
I had `info` here because we may WANT to shut down an RLS process. When it gets deleted, restarted etc.

Not necessarily an error.

Reply:
Gotcha. switching back